endpointmanager/pkg/archivefile/archivefile.go

```diff
@@ -17,21 +17,22 @@ import (
 
 // totalSummary is the format of a given URL's JSON object for the archive file
 type totalSummary struct {
-	URL                string                 `json:"url"`
-	CreatedAt          time.Time              `json:"created_at"`
-	ListSource         []string               `json:"list_source"`
-	OrganizationNames  []string               `json:"api_information_source_name"`
-	Updated            map[string]interface{} `json:"updated_at"`
-	NumberOfUpdates    int                    `json:"number_of_updates"`
-	Operation          map[string]interface{} `json:"operation"`
-	FHIRVersion        map[string]interface{} `json:"fhir_version"`
-	TLSVersion         map[string]interface{} `json:"tls_version"`
-	MIMETypes          firstLastStrArr        `json:"mime_types"`
-	Vendor             map[string]interface{} `json:"certified_api_developer_name"`
-	ResponseTimeSecond interface{}            `json:"median_response_time"`
-	HTTPResponse       []httpResponse         `json:"http_response"`
-	SmartHTTPResponse  []smartHTTPResponse    `json:"smart_http_response"`
-	Errors             []responseErrors       `json:"errors"`
+	URL                  string                 `json:"url"`
+	RequestedFhirVersion string                 `json:"requested_fhir_version"`
```

Comment:
I asked Jeff about this and he said that if there is a way to tell that the entries did not have the same requested version, he thought it would be valuable to keep both of the endpoints in the archive file!

Reply:
Okay sounds good, we can keep it as is then!